docs/content/latest/releases/releases-overview.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Supported and planned releases
+headerTitle: Supported and planned releases
+linkTitle: Supported and planned releases
+description: An overview of the latest and current stable releases, and planned releases.
+section: RELEASES
+menu:
+  latest:
+    identifier: releases-overview
+    weight: 2588 
+isTocNested: true
+showAsideToc: true
+---
+
+{{< note title="Important" >}}
+
+As of version `2.2`, Yugabyte is following a new release versioning convention for YugabyteDB releases.  For details, see [Release versioning](../versioning).
+
+{{< /note >}}
+
+## Supported stable releases
+
+The following stable release series are supported for production deployments. Starting with
+version `2.2`, Yugabyte is following a new [release versioning](../versioning) convention.
+The next stable release series is expected to be available in November 2020.
+
+**Note:** Versions `2.1`, `2.0`, and `1.3` that were released before `2.2` are also considered stable
+releases and are supported for existing production deployments.
+
+| Version | Released |
+| :------ | -------- |
+| 2.4                                    | 2020/11 (expected) |
+| [2.2](../earlier-releases/v2.2.0)      | 2020/07/15 |
+| [2.1](../earlier-releases/v2.1.0)      | 2020/02/25 |
+| [2.0](../earlier-releases/v2.0.0)      | 2019/09/17 |
+| [1.3](../earlier-releases/v1.3.0)      | 2019/07/19 |
+
+## Latest releases
+
+The latest release series specified below is a work-in-progress and introduces
+new features and changes that may break backward compatibility. Use the latest releases
```

Comment:
@stevebang I wouldn't call "changes that may break backward compatibility" anywhere instead say it's not recommended for any production use cases and it's subjected to changes. This will need changes in multiple places

Reply:
Updated all instances to remove that phrase.